src/Stratis.Bitcoin.Features.PoA/FederationMember.cs

```diff
@@ -66,8 +66,8 @@ public override int GetHashCode()
     /// <summary>Class that contains data that defines a federation member on federated peg sidechain.</summary>
     public class CollateralFederationMember : FederationMember
     {
-        public const decimal MinerCollateralAmount = 10_000m;
-        public const decimal MultisigMinerCollateralAmount = 50_000m;
+        public const decimal MinerCollateralAmount = 100_000m;
+        public const decimal MultisigMinerCollateralAmount = 100_000m;
```

Comment:
Hmm, why is the Multisig Collateral 100k? This should still be 50K...

Reply:
@trunghieud As discussed in discord, there are only for the new master node mining members. The existing members and their collateral requirements will be loaded elsewhere.